components/self_hosted_purchase_modal/seats_calculator.tsx

```diff
@@ -67,11 +68,17 @@ function validateSeats(seats: string, annualPricePerSeat: number, minSeats: numb
             }}
         />
     );
+
+    let errorPrefix = 'Self-serve';
+    if (cloud) {
+        errorPrefix = 'Cloud';
+    }
     const tooManyUsersErrorMessage = (
         <FormattedMessage
             id='self_hosted_signup.error_max_seats'
-            defaultMessage='Self-serve license purchase only supports purchases up to {num} users'
+            defaultMessage='{prefix} license purchase only supports purchases up to {num} users'
```

Comment:
I think we'll want two full, largely duplicated strings so that we can get full translation. Otherwise, in other languages we'll get things like `Self-Serve slizenzkauf unterstützt nur Käufe bis zu {num} Benutzer` and `Cloud slizenzkauf unterstützt nur Käufe bis zu {num} Benutzer`.